sql/conseil.sql

```diff
@@ -380,9 +380,6 @@ CREATE TABLE tezos.big_map_contents (
     PRIMARY KEY (big_map_id, key)
 );
 
-ALTER TABLE ONLY tezos.big_map_contents
```

Comment:
How come `Tables.scala` did not change?

Reply:
good one! 
weird, indeed... checking out